.gitignore

```diff
@@ -8,3 +8,4 @@ npm-debug.log
 /firefox
 lerna-debug.log
 flow-coverage
+.node-version
```

Comment:
I have multiple nodejs projects with multiple versions. This helps me keep a specific version per project. If you wish I can remove it but I don't think it bothers anybody. It contains a one line string with the node version v6.9.2